python/dask_cudf/dask_cudf/accessors.py

```diff
@@ -37,6 +37,32 @@ def field(self, key):
             meta=self.d_series._meta._constructor([], dtype=typ),
         )
 
+    def explode(self):
+        """
+        Creates a dataframe view of the struct column, one column per field.
+
+        Returns
+        -------
+        DataFrame
+
+        Examples
+        --------
+        >>> import cudf, dask_cudf as dgd
+        >>> ds = dgd.from_cudf(cudf.Series(
```

Comment:
Just a nitpick, but could we just go with `dask_cudf` instead of creating the `dgd` alias here?